### PR TITLE
Fix docker frontend update notification UI

### LINKED ISSUE #5086: Make the docker frontend update notification to a blocking full-screen page
## Problem Definition

User's might ignore it and try to login to a newer version of the backend that has breaking schema changes and boom.

If the frontend is outdated don't allow the user to use it until they update.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #5086

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
So @lukicenturi does Kelsos need to review this PR? Isnt' there any way for us to check that the fix does what it should do?

**Comment by lukicenturi** [comment]
> So @lukicenturi does Kelsos need to review this PR? Isnt' there any way for us to check that the fix does what it should do?

Yeah I think Kelsos need to review this

File: frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts [hunk 1]
```diff
@@ -91,8 +91,16 @@ describe('blockchain balances', () => {
...
       dashboardPage.getBlockchainBalances().then($dashboardBalances => {
-        expect($dashboardBalances).to.deep.eq($blockchainBalances);
+        $dashboardBalances.forEach((dashboardBalances, index) => {
+          expect(
+            dashboardBalances.renderedValue
+              .minus($blockchainBalances[index].renderedValue)
+              .abs()
+              .isLessThan(0.01)
+          );
+        });
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts:103]: I believe it has something to do with the rounding for the usdValue (price * amount).
We set the main currency to `EUR` in the test, and input an entry with `EUR` asset.
The value on the dashboard is rounded, by the value on the manual balances summary is not.

[the remark this answers is not in the document]
Comment (kelsos) [reply frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts:103]: Ok, that should not be the case though, at least without refreshing the prices, both tables should show the same thing and they should not have different numbers.

Do we have any video from Cypress on a test failure for this? to check the ongoing requests?

**Comment by kelsos** [comment]
@lukicenturi the notification part looks OK I am gonna rebase on top of develop and add a comment to figure out the mismatching values.

**Review by kelsos — APPROVED** [review]
lgtm

File: frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts]: we didn't compare value on tables to value on tables, but to value on summary card

[the remark this answers is not in the document]
Comment (kelsos) [reply frontend/app/tests/e2e/specs/accounts-balances/blockchain-balances.spec.ts]: > we didn't compare value on tables to value on tables, but to value on summary card

Ok, but still interesting that this didn't happen every single time but occasionally.

**Pull request merged**